cms/envs/common.py

```diff
@@ -461,11 +461,13 @@
 # List of finder classes that know how to find static files in various locations.
 # Note: the pipeline finder is included to be able to discover optimized files
 STATICFILES_FINDERS = [
-    'staticfiles.finders.FileSystemFinder',
-    'staticfiles.finders.AppDirectoriesFinder',
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     'pipeline.finders.PipelineFinder',
 ]
 
+PIPELINE_ENABLED = True
```

Comment:
I'm wondering whether this should be false by default. IMO it is easier to reason about the pipeline if settings that want optimization have to enable it, rather than having them all on by default in common and then having to disable them in other settings files. Having said that, when I started moving in this direction with https://github.com/edx/edx-platform/pull/9059, @feanil pointed out that this was breaking conventions we have for aws.py (see his comments in that PR).

If we stick with the current approach of having everything on in common, then we should probably re-enable the pipeline compression in this file, and then disable it in the places we don't want it.